resources/windows/wix/sequences.wxs.erb

```diff
@@ -14,7 +14,9 @@
         CMDLINE_INSTALLDIR
       </Custom>
       <Custom Action='SetIniPropertyValues' After='AppSearch' />
-      <Custom Action='GetLocalisedAccountNames' After='AppSearch' />
+      <Custom Action='GetLocalisedAccountNames' After='AppSearch'>
```

Comment:
```execute=immediate``` is the default - and is probably the best mode to run it in.
Will set this explicitly though (should really be set elsewhere too).

Do you think its ok to lift it completely from the UI sequence - its not needed there (whereas the IniProperties is probably needed in both sequences).